compiler/src/org.graalvm.compiler.serviceprovider.jdk11/src/org/graalvm/compiler/serviceprovider/GraalServices.java

```diff
@@ -437,4 +437,8 @@ public static VirtualObject createVirtualObject(ResolvedJavaType type, int id, b
         }
         return VirtualObject.get(type, id);
     }
+
+    public static int getJavaUpdateVersion() {
+        return Runtime.version().update();
```

Comment:
@dougxc Not sure if you rather wanted `Runtime.Version.parse(System.getProperty("java.vm.version")).update()` here. Let me know if so.

Reply:
No, this is exactly what we want. If `Runtime.Version.parse(System.getProperty("java.vm.version")).update() != Runtime.version().update()`, then I'd say there are bigger problems.